src/components/dropdown/bl-dropdown.ts

```diff
@@ -199,9 +147,9 @@ export default class BlDropdown extends LitElement {
       >
         ${this.label}
       </bl-button>
-      <div class="${popoverClasses}" role="menu" aria-expanded="${this.opened}">
+      <bl-popover .target="${this}" placement="bottom-start" @bl-popover-hide="${this.close}">
```

Comment:
Target should be the button above, not dropdown component itself.

Reply:
When I try to pass the target as the query result of the button, the popover component logs a warning because the button is not yet rendered, and the query result is null at initial render.

I checked the tooltip implementation PR, and it looks like the target is set on the popover with a show callback. We can implement it that way, if that's ok.

Reply:
Maybe it's time to try [`ref` directive](https://lit.dev/docs/templates/directives/#ref) of Lit, which is built for that kind of purposes. If it doesn't help, setting this lazily either in `firstUpdated` or just before showing it would also work.

Reply:
LGTM. Please make your PR ready for review.